custom_theme/community.html

```diff
@@ -57,30 +57,19 @@ <h1><img class="support-heading" src="../img/support-header.svg" alt="Life Prese
           <div class="col">
             <div class="card">
               <div class="card-body card-zendesk">
-                <h2>Open a Support Ticket! <span class="badge rounded-pill bg-success">NEW</span></h2>
-                <p>The easiest way to track your conversation and contribute to our knowledge base! Open a support ticket to begin talking to our support staff. Common questions will be added to our knowledge base to assist future Trufflers!</p>
-                <a target="_blank" rel="noopener noreferrer" href="https://trufflesuite.zendesk.com/hc/en-us/requests/new" class="btn btn-sm btn-truffle text-uppercase">Open a Support Ticket</a>
+                <h2>Join The Conversation</h2>
```

Comment:
Thoughts on changing this to "Join the Discussion" to be more inline with the fact it's Github Discussions?

Reply:
Agree. After syncing with @emilyJLin95, I did a revamp of the page hoping for a more simplified and organized look.